src/radix_connect/mobile/session/session.rs

```diff
@@ -2,7 +2,7 @@ use super::session_id::SessionID;
 use super::session_origin::SessionOrigin;
 use crate::prelude::*;
 
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
```

Comment:
I think `Session` can be `Copy` too, and then run `cargo clippy --fix --allow-staged`

Reply:
Oh never mind, the `SessionOrigin` is not copy... and cannot due to `Url` not being copy...